connectors/es/index.py

```diff
@@ -72,6 +72,7 @@ async def clean_index(self):
             index=self.index_name,
             body={"query": {"match_all": {}}},
             ignore_unavailable=True,
+            conflicts="proceed",
```

Comment:
Bonus points if you add a unit test

Reply:
Do you have a pattern of testing in mind? I can't think of reliable way of testing it, except for checking that function was called with an arg

Reply:
ah shoot that's right, we don't run tests with a real ES. :/ 
OK yeah, probably not super valuable to write a test that validates mocks.